plugins/languages/markdown/glow.nix

```diff
@@ -0,0 +1,96 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "glow";
+  originalName = "glow.nvim";
+  defaultPackage = pkgs.vimPlugins.glow-nvim;
+
+  maintainers = [ lib.maintainers.getchoo ];
+
+  settingsOptions = {
+    glow_path = helpers.defaultNullOpts.mkStr (lib.literalExpression "vim.fn.exepath(\"glow\")") ''
+      Path to `glow` binary
+
+      Consider using `glowPackage` instead
+    '';
+
+    install_path = helpers.defaultNullOpts.mkStr "~/.local/bin" ''
+      Path for installing `glow` binary if one is not found
+
+      Consider using `glowPackage` instead
+    '';
+
+    border =
+      helpers.defaultNullOpts.mkEnumFirstDefault
+        [
+          "shadow"
+          "none"
+          "double"
+          "rounded"
+          "solid"
+          "single"
+        ]
+        ''
+          Style of the floating window's border
+        '';
```

Comment:
I think nixfmt may format this nicer if you use a `"` string.

We do _normally_ prefer `''`-indented strings for option descriptions, but in this case I think `"` would be better.